apps/chat-e2e/src/ui/pages/dialHomePage.ts

```diff
@@ -32,9 +32,15 @@ export class DialHomePage extends BasePage {
       state: 'hidden',
       timeout: loadingTimeout,
     });
-    await appContainer
-      .getChatLoader()
-      .waitForState({ state: 'hidden', timeout: loadingTimeout });
+    //workaround for the issue https://github.com/epam/ai-dial-chat/issues/1596
+    try {
+      await appContainer
+        .getChatLoader()
+        .waitForState({ state: 'hidden', timeout: loadingTimeout });
+    } catch (error) {
+      await this.reloadPage();
```

Comment:
We are going to reload it only once? Is that enough?

Reply:
the method is called recursevely in the next line